service/manager.go

```diff
@@ -85,6 +86,10 @@ func newManager(builder *Builder) (Manager, error) {
 		return nil, err
 	}
 
+	if err := m.configProvider.Get(config.Root).Populate(m); err != nil {
```

Comment:
they would all have to share the timeout then? that's ok for the time being if that's not a popular use case.